tasmota/tasmota_xdrv_driver/xdrv_01_9_webserver.ino

```diff
@@ -351,6 +351,7 @@ const char HTTP_FORM_OTHER[] PROGMEM =
   "<br>"
   "<label><input id='b3' type='checkbox'%s><b>" D_HTTP_API_ENABLE "</b></label><br>"
   "<label><input id='b1' type='checkbox'%s><b>" D_MQTT_ENABLE "</b></label><br>"
+  "<label><input id='b4' type='checkbox'%s><b>" D_MDNS_ENABLE "</b></label><br>"
```

Comment:
This must be guarded by `#ifdef USE_DISCOVERY` to not show it when MDNS is not present in the build